lib/utilities/get-script-file-name.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function getScriptFileName(filePath) {
+  const scriptFileNameRegEx = /([a-zA-Z0-9_\.\-\(\):])+(\.js)/ig;
```

Comment:
Shouldn't this be a constant outside the function so it is only compiled once?

Reply:
If memory serves regexe's with `/g` have state